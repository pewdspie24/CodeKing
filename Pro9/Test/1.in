10
751347 976003
564166 753736
144050 230040
917571 967950
985768 994145
620828 734717
294609 655998
610450 831774
127943 750028
211032 922803
