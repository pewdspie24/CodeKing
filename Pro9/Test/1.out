4011
4042
3608
792
148
2645
10786
4712
18985
18996
